types/braintree/index.d.ts

```diff
@@ -94,14 +102,14 @@ declare namespace braintree {
     }
 
     interface CreditCardVerificationGateway {
-        search(searchFn: any): Promise<CreditCardVerification[]>;
+        search(searchFn: any): stream.Readable;
```

Comment:
@sharno Thanks for fixing search and adding in streams in. Do you know if there is there a way to add typings to the stream objects (i.e. `CreditCardVerification` here)? That could be very nice to have.

Reply:
I tried to look for something like this but I couldn't find any. It's much helpful of course if we could type the stream but it doesn't look the be the case in `@types/node`.

A solution might be looking into if it's possible to make `@types/node`'s `stream.Readable` take a type param.

A custom stream type for braintree is possible but wouldn't work well with the other stream types from `@types/node`